apps/expo/src/app/activity/feed.tsx

```diff
@@ -0,0 +1,40 @@
+import { Text } from 'react-native'
+import { FlatList } from 'react-native-gesture-handler'
+import { SafeAreaView } from 'react-native-safe-area-context'
+
+import { useGlobalContext } from '~/context/global-context'
+import { api } from '~/utils/trpc/api'
+import Activity from '.'
+import { ACTIVITY_FEED_ITEM_LIMIT } from '../../utils/constants'
+
+const ActivityFeed = () => {
+  let activities: any[] = []
+  const { userData } = useGlobalContext()
+  const { data: friends, isLoading: friendsIsLoading } = api.user.getFriends.useQuery({
+    userId: userData?.id ?? 0,
+  })
+  const { data: friendsActivities, isLoading: friendsActivitiesLoading } =
+    api.workout.getActivityFeedWorkouts.useQuery(
+      {
+        friendIds: friends?.map((friend) => friend.friendId) ?? [],
+        count: ACTIVITY_FEED_ITEM_LIMIT,
+      },
+      { enabled: !friendsIsLoading },
```

Comment:
trpc convention for the `enabled` query you gotta use !! operator for more edge case handling. You can make a commit to update, or nein, as it should work for this particular query ...

Reply:
Well `!!` converts a truthy/falsey value to `true`/`false`. I want the opposite of that, so `!` should be sufficient.

Reply:
if its logic then great, I overlooked it